server/index.js

```diff
@@ -18,6 +21,21 @@ app.get('/api/hello', (req, res) => {
 
 app.use(errorMiddleware);
 
```

Comment:
This should go right before `.listen()` so that errors are handled by the error middleware.